pkg/types/hashedrekord/v0.0.1/entry_test.go

```diff
@@ -297,9 +304,15 @@ func TestCrossFieldValidation(t *testing.T) {
 			if err != nil {
 				t.Errorf("unexpected err from Unmarshalling canonicalized entry for '%v': %v", tc.caseDesc, err)
 			}
-			if _, err := types.UnmarshalEntry(pe); err != nil {
+			ei, err := types.UnmarshalEntry(pe)
+			if err != nil {
 				t.Errorf("unexpected err from type-specific unmarshalling for '%v': %v", tc.caseDesc, err)
 			}
+			// hashedrekord is one of two types (rfc3161, hashedrekord) in that what is persisted is also insertable
```

Comment:
What do you mean by this?

Reply:
Proposed entries are potentially unmodified when canonicalized, so what is proposed may be exactly what is persisted. RFC3161 timestamps are also persisted as written, so the notion of “ensuring proposed entries are insertable but canonicalized entries are not” makes sense for some pluggable types but not others.

Reply:
Makes sense, thanks